src/DIPS.Xamarin.UI/Controls/Toast/ToastLayout.cs

```diff
@@ -0,0 +1,79 @@
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Toast
+{
+    /// <summary>
+    ///     Set layout options for the Toast control
+    /// </summary>
+    public class ToastLayout : BindableObject
```

Comment:
Does this need to be a `BindableObject`? The consumer get bloated with a lot of properties / methods that they don't care about.

Reply:
Oh, same residue... missed to remove the extend after removing the bindings